app/views/maps/show.html.erb

```diff
@@ -11,7 +11,7 @@
   <div class="row">
     <div class="col-12">
       <div class="pre-scrollable map-view">
-        <%= image_tag asset_pack_path("media/images/maps/#{@map.background}") %>
+        <div id="map" class="w-100 h-100" <%= data_attributes(@map) %>></div>
```

Comment:
Can WebSockets or ajax be used to transfer this data instead? Would that be better handled in a separate task?

Reply:
https://github.com/manik1235/story_time/issues/67